helpdesk_mgmt_timesheet_restricted/views/helpdesk_mgmt_timesheet_restricted.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_helpdesk_ticket_timesheet_restricted_view_tree_inherit" model="ir.ui.view">
+        <field name="name">helpdesk.ticket.view.tree (in helpdesk_mgmt_timesheet_restricted)</field>
+        <field name="model">helpdesk.ticket</field>
+        <field name="inherit_id" ref="helpdesk_mgmt_timesheet.ticket_view_tree"/>
+        <field name="groups_id" eval="[(4,ref('group_helpdesk_timesheet_restricted'))]"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='planned_hours']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='total_hours']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='remaining_hours']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_helpdesk_ticket_timesheet_restricted_view_form_inherit" model="ir.ui.view">
+        <field name="name">helpdesk.ticket.view.form (in helpdesk_mgmt_timesheet_restricted)</field>
+        <field name="model">helpdesk.ticket</field>
+        <field name="inherit_id" ref="helpdesk_mgmt_timesheet.timesheet_helpdesk_ticket_view_form"/>
+        <field name="groups_id" eval="[(4,ref('group_helpdesk_timesheet_restricted'))]"/>
+        <field name="arch" type="xml">
+            <xpath expr="//page/group" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
```

Comment:
Unfortunately, page element on parent view has no name attribute. Selecting without attributes is dangerous, so I prefer this:
```suggestion
            <xpath expr="//field[@name='planned_hours']" position="attributes">
                <attribute name="invisible">1</attribute>
            </xpath>
            <xpath expr="//field[@name='progress']" position="attributes">
                <attribute name="invisible">1</attribute>
            </xpath>
```
It's more code and result is slightly different, but more secure.

Reply:
@dalonsod You are right, my first solution was that, but there is an empty space that I did not like. But if you like me too.
![imagen](https://user-images.githubusercontent.com/22427989/112116135-3d169480-8bba-11eb-8535-21c95b7a95c3.png)

Reply:
I'm aware, please do it